src/20240707_AaveV3Ethereum_ToolingUpdateAllowance/ToolingUpdateAllowance.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Tooling Update Allowance"
+author: "TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-treasury-management-tooling-upgrade/15201"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x461571b38365b12ebe39b80d4d9663daa9c7e574cd4bf190ec6fb48dec96371f"
+---
+
+## Simple Summary
+
+As per the forum vote that contracted TokenLogic to develop various smart contracts to help with treasury management by bridging funds from multiple networks, after having the DAO utilize these solutions we are seeking payment for the work performed.
+
+## Motivation
+
+To ensure the Aave DAO's financial stability and the ability to transfer assets across different networks through the shortExecutor, it is crucial to establish equivalent contracts for other networks. One such scenario involves moving USDC from Avalanche to Ethereum, effectively extending the DAO's stable coin runway.
+
+## Specification
+
+Set an allowance of 12,000 GHO to TokenLogic.
+
+## References
```

Comment:
Could you provide github links to the work done?

The forum does not show progress on this work, nor do the results appear here. It will be quite inconvenient to search.

Reply:
We added a comment on the forum, linked [here](https://governance.aave.com/t/arfc-treasury-management-tooling-upgrade/15201/5) for easy reference.